capstone/node_modules/@types/bluebird/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
I think you'll want to add `capstone/node_modules` to a .gitignore file (at the top directory of this github repo).  That will tell git not to add these files to our repo, since they aren't really part of are code base and are already hosted elsewhere.

I think we'll probably want to update the README.md with instructions for how to install the dependencies.  Typically most JS/TS repos have a file called `package.json` which lists all the npm dependencies it needs, and users will run a command like `npm install` to install all packages listed in it (I haven't used npm in a few years so my commands might not be 100% accurate, @lgsds is probably the expert here)

Reply:
+1 for .gitignore, like [so](https://github.com/conversationai/perspectiveapi-authorship-demo/blob/master/.gitignore#L33).

Also +1 to updating the README with the relevant instructions for installing the dependencies and running everything.

Reply:
I have added a package.json file and I will soon update the README.md accordingly.